test-clients/src/main/java/com/hedera/services/bdd/spec/queries/contract/HapiContractCallLocal.java

```diff
@@ -113,6 +119,22 @@ public HapiContractCallLocal saveResultTo(String key) {
 		return this;
 	}
 
+	public HapiContractCallLocal withAccessedKeys(ContractID contractID, UInt256... keys) {
```

Comment:
I think that we should rename **both** methods to `withAccessList`. I think that it fits better since we may include only a contract ID/account ID and not keys.